src/steamcompmgr.cpp

```diff
@@ -3397,6 +3492,9 @@ steamcompmgr_main (int argc, char **argv)
 			case 'x':
 				useXRes = False;
 				break;
+			case 'a':
+				g_customCursorPath = strdup(optarg);
```

Comment:
No need to `strdup` here, `optarg` is valid for the lifetime of the program.

Reply:
Hooray!